src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -917,7 +927,7 @@ private CloseableHttpClient getHttpClient(final HttpRequestBase request) {
     }
 
     private void setClientProxyParams(String host, HttpClientBuilder builder) {
-        Jenkins jenkins = Jenkins.get();
+        Jenkins jenkins = Jenkins.getInstanceOrNull();
```

Comment:
there should be at no point Jenkins can be null so not sure the if jenkins is null check should be theere.